### PR TITLE
Fixed few bugs related to signOut and sessions

### PR DESCRIPTION
**Pull request by Comaecod**
https://anujsingh.atlassian.net/browse/NEX-28
https://anujsingh.atlassian.net/browse/NEX-29
https://anujsingh.atlassian.net/browse/NEX-30

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "rules": {
     "no-console": "error",
     "no-use-before-define": "error",
-    "no-unused-vars": "error",
+    "no-unused-vars": "off",
```
Comment (anujSingh-A-writer) [inline .eslintrc.json]: Make it `error `

**Review by anujSingh-A-writer — CHANGES_REQUESTED** [review]
Great work

File: .eslintrc.json (hunk 1 above)
Comment (anujSingh-A-writer) [reply .eslintrc.json]: Done

Comment (Comaecod) [reply .eslintrc.json]: Done.

**Pull request merged**